io/plugins/eu.esdihumboldt.hale.io.csv/src/eu/esdihumboldt/hale/io/csv/reader/internal/CSVConfiguration.java

```diff
@@ -85,8 +85,8 @@ public char getEscape() {
 	/**
 	 * @return the skip_first_line
```

Comment:
As the parameter `skip_first_line` no longer exists, in the comment
 `@return skipNlines` instead of `@return the skip_first_line`?